src/esi/Expression.cc

```diff
@@ -32,7 +32,7 @@
  * ) has 0
  */
 
-typedef struct _stackmember stackmember;
+struct stackmember;
```

Comment:
Is it a showstopper? current trunk feels old-style, and it feels silly to have a PR for that only.

Reply:
> Is it a showstopper?

IMO, no. I am not blocking this PR.

> current trunk feels old-style, and it feels silly to have a PR for that only

You are correct on both counts, but neither observation justifies merging these out-of-scope changes:

* PR changes in question do not follow the current Squid code styling guidelines. They attempt to update the old-style code, but they are themselves problematic.

* There are hundreds of similar lines that need a style update. Such an update can and should be automated. We encourage (and some of us even require!) style updates for lines _modified_ due to in-scope changes (because doing so often does not complicate diff comprehension and often does not increase merge conflicts -- the diff already contains the changed lines), but these lines do qualify for that special exception.

* A PR should not have out of scope changes, for several obvious/basic reasons. A reviewer should not be put in the position where they have to defend the request to remove out of scope changes. If there are no special circumstances (none have been provided so far), out of scope changes should be removed.